index.html

```diff
@@ -925,7 +940,28 @@ <h2>Yusuf Pisan</h2>
 				</div>
 			</div>
 		</div>
+
 		<div class="md-overlay"></div>
+
+		<div class="md-modal md-effect-9" id="modal-10">
+			<div class="md-content">
+				<div class="folio">
+					<div class="avatar10"></div>
+					<div class="sp-name"><strong>Nikhil Alapati</strong><br />General Officer</div>
+					<div class="sp-dsc">
+						Nikhil Alapati is a Sophtmore Computer Science student at UWB. He is interest in Game Developement and playing video games. Husky Game Dev intrests him because he wants to bring out the fullest potential of UWB students 

```

Comment:
```suggestion
						Nikhil Alapati is a Sophomore Computer Science student at UWB. He is interested in Game Development and playing video games. Husky Game Dev interests him because he wants to bring out the fullest potential of UWB students.

```

Just a few typos I figured you'd want to fix. :)